### PR TITLE
Add contributor writeup

### PR DESCRIPTION
**Pull request by amyrlam**
- [x] @ijlee2 and @jaredgalanis please review this link if you have time: https://discuss.emberjs.com/t/i-contribute-to-ember-with-isaac-lee-and-jared-galanis/18127

### CODE REVIEW

File: source/2020-08-14-the-ember-times-issue-160.md [hunk 1]
```diff
@@ -9,21 +9,36 @@ responsive: true
...
-## [Section title in sentence case 🐹](section-url)
+## ["I contribute to Ember" with Isaac Lee and Jared Galanis 🙌](https://discuss.emberjs.com/t/i-contribute-to-ember-with-isaac-lee-and-jared-galanis/18127)
 
-<change section title emoji>
 <consider adding some bold to your paragraph>
 <please include link to external article/repo/etc in paragraph / body text, not just header title above>
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<div class="float-right padded portrait-frame">
```
Comment (amyrlam) [inline source/2020-08-14-the-ember-times-issue-160.md:25]: I'm pretty sure the 2x image markup won't work as-is, but putting in as a placeholder

File: source/2020-08-14-the-ember-times-issue-160.md [hunk 2]
```diff
@@ -9,21 +9,36 @@ responsive: true
...
+  <img alt="Isaac Lee" title="Isaac Lee - Contributor to Ember" src="/images/blog/emberjstimes/isaac_lee.jpg" />
+</div>
+
+<div class="float-right padded portrait-frame">
+  <img alt="Jared Galanis" title="Jared Galanis - Contributor to Ember" src="/images/blog/emberjstimes/jared_galanis.jpg" />
+</div>
+
+In the **fifteenth edition** of our contributor interview series, we're highlighting The Ember Times' very own [Isaac Lee (@ijlee2)](https://github.com/ijlee2) and [Jared Galanis (@jaredgalanis)](https://github.com/jaredgalanis). We'd like to thank Isaac and Jared for their contributions to the Ember community, and congratulate them for joining the Ember Learning Core Team!
+
+TODO: add teaser blurb!
```
Comment (amyrlam) [inline source/2020-08-14-the-ember-times-issue-160.md:35]: Haven't added this yet, but wanted to get this up so there is time for you to review / also feel free to suggest copy, but no pressure!

File: source/2020-08-14-the-ember-times-issue-160.md [hunk 3]
```diff
@@ -9,21 +9,36 @@ responsive: true
...
-<add blurb and emoji to "SOME-INTRO-HERE">
+<div class="float-right padded portrait-frame">
+  <img alt="Isaac Lee" title="Isaac Lee - Contributor to Ember" src="/images/blog/emberjstimes/isaac_lee.jpg" />
+</div>
+
+<div class="float-right padded portrait-frame">
+  <img alt="Jared Galanis" title="Jared Galanis - Contributor to Ember" src="/images/blog/emberjstimes/jared_galanis.jpg" />
+</div>
+
+In the **fifteenth edition** of our contributor interview series, we're highlighting The Ember Times' very own [Isaac Lee (@ijlee2)](https://github.com/ijlee2) and [Jared Galanis (@jaredgalanis)](https://github.com/jaredgalanis). We'd like to thank Isaac and Jared for their contributions to the Ember community, and congratulate them for joining the Ember Learning Core Team!
```
Comment (ijlee2) [inline source/2020-08-14-the-ember-times-issue-160.md]: For simplicity (for future editions), I think we could start writing the number. 🙂

```suggestion
In the **15th edition** of our contributor interview series, we're highlighting The Ember Times' very own [Isaac Lee (@ijlee2)](https://github.com/ijlee2) and [Jared Galanis (@jaredgalanis)](https://github.com/jaredgalanis). We'd like to thank Isaac and Jared for their contributions to the Ember community, and congratulate them for joining the Ember Learning Core Team!
```

**Review by ijlee2 — APPROVED** [review]
Looks good!

File: source/2020-08-14-the-ember-times-issue-160.md [hunk 4]
```diff
@@ -9,21 +9,36 @@ responsive: true
...
+
+TODO: add teaser blurb!
+
+Check out the full interview on [the Ember Discuss forum](https://discuss.emberjs.com/t/i-contribute-to-ember-with-isaac-lee-and-jared-galanis/18127).
+
+<a class="ember-button ember-button--centered" href="https://discuss.emberjs.com/t/i-contribute-to-ember-with-isaac-lee-and-jared-galanis/18127">Read more</a>
+
+<p style="font-style: italic;">The idea behind these interviews is to provide community members with more information and insight into open source contributions, learning resources and community events. Contributors to the community gain a platform to share their thoughts on their own experiences and learnings as they were contributing to Ember. Our hope is that it gives first-time and advancing contributors a better sense of the fact that everyone starts out small, and that anyone can have a big impact on the community.</p>
```
Comment (abhilashlr) [inline source/2020-08-14-the-ember-times-issue-160.md]:
```suggestion
<p style="font-style: italic;">The idea behind these interviews is to provide community members with more information and insight into open source contributions, learning resources, and community events. Contributors to the community gain a platform to share their thoughts on their own experiences and learnings as they were contributing to Ember. Our hope is that it gives first-time and advancing contributors a better sense of the fact that everyone starts out small, and that anyone can have a big impact on the community.</p>
```

Comment (abhilashlr) [reply source/2020-08-14-the-ember-times-issue-160.md]: I added a `,` in the first sentence.

`
The idea behind these interviews is to provide community members with more information and insight into open source contributions, learning resources, and community events. 
`

Feel free to discard.

**Review by abhilashlr — APPROVED** [review]
The write reads well @amyrlam 👍

File: source/2020-08-14-the-ember-times-issue-160.md (hunk 3 above)
Comment (amyrlam) [reply source/2020-08-14-the-ember-times-issue-160.md]: sure!

File: source/2020-08-14-the-ember-times-issue-160.md (hunk 4 above)
Comment (amyrlam) [reply source/2020-08-14-the-ember-times-issue-160.md]: sgtm!

**Comment by amyrlam** [comment]
i'll make updates to this later today. verbiage extracted from the discuss, and playing around with it locally for the images

**Review by jaredgalanis — APPROVED** [review]
Looks great, thank you 💖

**Comment by amyrlam** [comment]
will make the edits upstream

**Pull request merged**